source/opt/def_use_manager.cpp

```diff
@@ -156,17 +195,8 @@ bool operator==(const DefUseManager& lhs, const DefUseManager& rhs) {
     return false;
   }
 
-  for (auto use : lhs.id_to_uses_) {
-    auto rhs_iter = rhs.id_to_uses_.find(use.first);
-    if (rhs_iter == rhs.id_to_uses_.end()) {
-      return false;
-    }
-    use.second.sort();
-    UseList rhs_uselist = rhs_iter->second;
-    rhs_uselist.sort();
-    if (use.second != rhs_uselist) {
-      return false;
-    }
+  if (lhs.id_to_users_ != rhs.id_to_users_) {
```

Comment:
Not for this CL, but .... Why do we need an equality comparison on a DefUseManager ???

Reply:
This is used in the consistency checks (in debug builds) to ensure analyses passes promise to preserve are actually correct.